src/Microsoft.AspNet.Mvc.DataAnnotations/NumericClientModelValidator.cs

```diff
@@ -16,11 +16,21 @@ public class NumericClientModelValidator : IClientModelValidator
         /// <inheritdoc />
         public IEnumerable<ModelClientValidationRule> GetClientValidationRules(ClientModelValidationContext context)
         {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
```

Comment:
Just curious: Is your compile module so smart that it nocied the `[NotNull` in the interface? Or did you catch this manually? I'm hoping the first of course...